common/components/forms/FormValidation.jsx

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import React from 'react';
+import _ from 'lodash'
+
+export type Validation = {|
+  checkFunc: (formState: any) => boolean,
```

Comment:
this perhaps could be renamed to `validator`

Reply:
Renamed